src/css/_header.scss

```diff
@@ -41,12 +41,54 @@ header {
   }
 }
 
+#city-dropdown {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
 #city-dropdown label {
   font-weight: normal;
+  padding-right: 20px;
 }
 
 #city-choice {
   border: 2px solid rgba(0, 0, 0, 0.2);
   border-radius: 10px;
   color: black;
 }
+
+.choices, .is-open {
```

Comment:
Did you mean for this to be `.choices.is-open`, or `.choices, .is-open`? If the latter, a more clear style is to have each selector on a distinct line.